extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php

```diff
@@ -34,6 +34,7 @@ public function getLogInfo() {
 			->warning( 'AdBlock Interference',
 				[ 'action' => $this->request->getVal('kind') ]
 			);
+		$this->response->setHeader('X-Robots-Tag', 'noindex,noarchive'); //ADEN-3930
 		$this->response->setContentType( 'text/javascript; charset=utf-8' );
 		$this->response->setBody( 'var arecoveryEngineLogInfoStatus=1;' );
 		$this->response->setCacheValidity( self::MAX_EVENT_INTERVAL );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)